.github/workflows/automated-test-acceptances.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Automated Acceptances Tests'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+      - name: Acceptance Tests
+        env:
+          SKIP_TEST_EXTERNAL_CREDENTIALS: ${{ secrets.SKIP_TEST_EXTERNAL_CREDENTIALS }}
```

Comment:
That's correct, by default the env `SKIP_TEST_EXTERNAL_CREDENTIALS` is false, so if you don't set the env if will run the test that requires external credentials, obviously if you don't set them it will appear error about credentials for aws/gcp/azure

Reply:
But won't we need a line to import them into the test env as we set them?   e.g.      AWS_VPC_CIDR_BLOCK: ${{ secrets.AWS_VPC_CIDR_BLOCK }}   Can't we pre-seed them all here for the default SKIP_TEST_EXTERNAL_CREDENTIALS=false but skip setting them in the env when SKIP_TEST_EXTERNAL_CREDENTIALS=true?

Reply:
Oh, you're right, I need to add those env var as well, I misunderstood you at first. Thank you for that, melissa